KMD.Identity.TestApplications.SAML.MVCCore/appsettings.json

```diff
@@ -7,15 +7,35 @@
     }
   },
   "Saml2": {
-    "IdPMetadata": "https://identity.kmd.dk/federationmetadata/2007-06/federationmetadata.xml",
```

Comment:
Should we maybe keep this and out comment it and leave a note about when to use which metadata?

Reply:
reverted to the previous